e2e/utils.sh

```diff
@@ -333,13 +277,7 @@ setup_actuallylocal_shared_network() {
 }
 
 setup_local_shared_network() {
-    local replica_port
-    if [ "$USE_IC_REF" ]
-    then
-        replica_port=$(get_ic_ref_port)
-    else
-        replica_port=$(get_replica_port)
-    fi
+    local replica_port=$(get_replica_port)
```

Comment:
This should check for IS_POCKET_IC and call either get_replica_port or a new function that reads `pocket_ic_port`

Reply:
Also dead code => removed.